doc/man3/OSSL_PROVIDER.pod

```diff
@@ -47,6 +48,8 @@ OSSL_PROVIDER_unload() unloads the given provider.
 For a provider added with OSSL_PROVIDER_add_builtin(), this simply
 runs its teardown function.
 
+OSSL_PROVIDER_available() checks if a named provider has been loaded.
```

Comment:
Can OPENSSL_CTX be NULL?

Reply:
What happens if no providers have been loaded yet, but this is a fallback provider?

Reply:
> Can OPENSSL_CTX be NULL?

Does this really need to be repeated *everywhere*?

> What happens if no providers have been loaded yet, but this is a fallback provider?

Will rephrase

Reply:
> Does this really need to be repeated everywhere?

Yes, I think so. Or if not then a reference to where it is stated should be given everywhere. These are man pages so we have to assume that people dip in and out of them without the context of having read other pages.

Reply:
I prefer a link to OPENSSL_CTX(3)

Reply:
Done

Reply:
Hmm. Just a link in the "See Also" section doesn't seem sufficient.

Reply:
Why not?  If someone wants to know more about OPENSSL_CTX, then they should read the manual page for OPENSSL_CTX, doesn't that make sense?

Reply:
If I was reading the man page for a function I would expect to see some discussion of all of the parameters for that function even if it is just to say "...see L<OPENSSL_CTX(3)> for further details of the B\<ctx\> parameter". The fact that a separate OPENSSL_CTX man page exists may not be immediately obvious, and having its reference buried away in the SEE ALSO at the very bottom of the page is likely to be missed.

Reply:
I added a general line in the general description.  I'm starting to think that's what we should do, generally, rather than have it attached to each function (which is its own kind of buried)